yt/frontends/ramses/hilbert.py

```diff
@@ -259,10 +95,50 @@ def hilbert3d(X, bit_length):
     return order
 
 
-def get_cpu_list(ds, X):
+def get_cpu_list(
+    ds, bbox: np.ndarray, LE: np.ndarray = None, dx: float = 1.0, dx_cond: float = None
+) -> set[int]:
+    """
+    Find the subset of CPUs that intersect the bbox in a recursive fashion.
+    """
+    if LE is None:
+        LE = np.array(
+            [
+                0,
+                0,
+                0,
+            ],
+            dtype="d",
+        )
```

Comment:
I think the trailing comma after the last "0" is causing this line to be expanded by black, removing it should stabilise styling and readability
```suggestion
        LE = np.array([0, 0, 0], dtype="d")
```